brian_cheung/week_one_challenges/string_manipulation.md

```diff
@@ -19,13 +19,27 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
 Programmatically manipulate `iverson_rant` to reflect the output above.
 
 ```ruby
-# write your code here
+def simplify (phrase)
+  abbr = nil
```

Comment:
MegaNitpick: In Ruby, you don't generally put a space between your method name and your arguments (e.g. `def simplify(phrase)` vs. `def simplify (phrase)`